src/libstd/sys/common/condvar.rs

```diff
@@ -18,14 +20,14 @@ use sys::condvar as imp;
 /// condition variables. It is consequently entirely unsafe to use. It is
 /// recommended to use the safer types at the top level of this crate instead of
 /// this type.
-pub struct Condvar(imp::Condvar);
+pub struct Condvar(Box<imp::Condvar>);
```

Comment:
@tmiasko yeah let's keep the boxes in`std::sync` to stay consistent with the mutex module next to this one.